### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by viru2001**
added homepage screen 

Screenshot 2022-04-01 200955

### CODE REVIEW

**Review by ark2002 — APPROVED** [review]
Ui looks great even the code is easy to read !

File: public/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/notes-favicon.png" />
```
Comment (ravikumar0403) [inline public/index.html]: You can use [favicon.io](https://favicon.io/) to convert png to ico

File: src/frontend/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import { Link, useLocation } from "react-router-dom";
+import { NotesIcon } from "../../assets";
```
Comment (ravikumar0403) [inline src/frontend/components/Navbar/Navbar.jsx]: you can use path alias to avoid relative paths

**Review by ravikumar0403 — APPROVED** [review]
Good folder structure and code is also clean

**Pull request merged**